### ISSUE TITLE
Issue deploying Storybook to GH Pages

### ISSUE THREAD

**Issue by jaebradley** [jaebradley/github-portfolio-component#14]
![image](https://user-images.githubusercontent.com/8136030/35741500-fc95cca4-0805-11e8-98d2-c74903bb8396.png)

**Comment by jaebradley** [comment]
Potentially relevant `gists`
* https://gist.github.com/domenic/ec8b0fc8ab45f39403dd
* https://gist.github.com/willprice/e07efd73fb7f13f917ea

**Comment by jaebradley** [comment]
This script does the trick

```bash
#! /bin/bash

rm -rf out || exit 0;
mkdir out;
node ./node_modules/.bin/build-storybook -o out;

cd out
echo 'initializing git'
git init
git config user.name 'jaebradley'
git config user.email 'jae.b.bradley@gmail.com'
echo 'adding files'
git add .
echo 'commiting files'
git commit -m 'Deployed to GH Pages'
git push --force --quiet "https://${GH_TOKEN}@github.com/jaebradley/github-portfolio-component" master:gh-pages > /dev/null 2>&1
```

**Comment by jaebradley** [comment]
https://github.com/blog/1270-easier-builds-and-deployments-using-git-over-https-and-oauth

**Comment by jaebradley** [comment]
Resolved by #15
